themes/coderdojoschoeneweide/layouts/robots.txt

```diff
@@ -0,0 +1,27 @@
+User-agent: *
+Disallow: /de/impressum
+Disallow: /datenschutz
```

Comment:
Hier fehlt noch das Language-Prefix :)
Wir haben die Seiten übrigens auch in Arabisch (/ar/datenschutz, /en/impressum) und demnächst in Englisch (/en/datenschutz, /en/impressum). Könntest Du die bitte auch gleich noch mit aufnehmen?